be/src/exec/pipeline/scan/scan_operator.cpp

```diff
@@ -217,8 +216,20 @@ void ScanOperator::_detach_chunk_sources() {
 
 Status ScanOperator::set_finishing(RuntimeState* state) {
     std::lock_guard guard(_task_mutex);
+    if (UNLIKELY(state != nullptr && state->query_ctx()->is_query_expired() &&
+                 (_num_running_io_tasks > 0 || _submit_task_counter->value() == 0 ||
+                  _peak_io_tasks_counter->value() == 0))) {
+        LOG(WARNING) << "set_finishing scan fragment " << print_id(state->fragment_instance_id()) << " driver_id  "
```

Comment:
prefer VLOG

Reply:
if without this log, we can't know the scan is running or not run any more. The profile may not enable or lost, the only way is to get some msg from log.
It only works when timeout to help us to fix bugs.

Reply:
can we turn on verbose flag in runtime when we need it ?